docs/guides/server/importExport.adoc

```diff
@@ -127,4 +127,38 @@ When importing a realm at startup, you are able to use placeholders to resolve v
 
 In the example above, the value set to the `MY_REALM_NAME` environment variable is going to be used to set the `realm` property.
 
-</@tmpl.guide>
\ No newline at end of file
+== Importing and Exporting via the Admin Console
+
+It is also possible to import and export a realm using {project_name}'s Admin Console. This functionality is
+different from the other CLI options presented above because the Admin Console offers only the capability to
+*partially* export a realm. In this case, the current realm settings, along with some resources like clients,
+roles, and groups, can be exported. The realm's users *cannot* be exported using this method.
```

Comment:
```suggestion
You can also import and export a realm using the Admin Console. This functionality is
different from the other CLI options described in previous sections because the Admin Console offers only the capability to
_partially_  export a realm. In this case, the current realm settings, along with some resources like clients,
roles, and groups, can be exported. The users for that realm _cannot_ be exported using this method.
```